integrations/prometheus/README.md

```diff
@@ -0,0 +1,11 @@
+# Prometheus Metrics
+
+This configuration file can be used to set up a Prometheus instance to read
+data from  Nomad cluster, using Consul for service discovery.
+
+Requirements:
+  - See Prometheus's
+    [Getting Started](https://prometheus.io/docs/introduction/getting_started/)
+    guide for instructions on how to set up a Prometheus server.
+  - A running Consul server, which will need to be added to this configuration
```

Comment:
Can you clarify this point. I think you may want to say that the configuration is setup to read from the local agent and thus if the prometheus server is run on a machine without a Consul agent, the configuration will have to be updated to point to a Consul server.